docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "2"
+services:
+  redis:
+    container_name: redis
+    image: redis
+    ports:
+      - 6379:6379
+    volumes:
+      - dev-env-redis-volume:/data
+  hmrc-postgres:
+    container_name: postgres
+    image: postgres:10
+    environment:
+      - PGDATA=/var/lib/postgresql/data/pgdata
+      - POSTGRES_USER=ale
```

Comment:
I think this can be `POSTGRES_USER=${USER}` so its the developers own username rather then hardcoding to `ale` - which should then work with the `database.yml` contents

Reply:
ops, yes `ale` must not be there! 😄